plugins/woocommerce/client/blocks/tests/e2e/tests/templates/template-customization.block_theme.spec.ts

```diff
@@ -133,7 +147,9 @@ test.describe( 'Template customization', () => {
 	} );
 
 	const testToRun = CUSTOMIZABLE_WC_TEMPLATES.filter(
-		( data ) => data.canBeOverriddenByThemes
+		( data ) =>
+			data.canBeOverriddenByThemes &&
+			data.templateType === 'wp_template_part'
```

Comment:
Do you know why we check that it's a template part here? Are we intentionally skipping regular templates?

Reply:
This is something we figured out together on a pairing session when [I was initially testing Template Activation](https://github.com/woocommerce/woocommerce/pull/61046/files#diff-e24a3a268bc59d1b941bc95b6a530f922e419df79c9972964287f3034798d30dR152). These tests no longer make sense with the activation since the hierarchy no longer takes place, you either activate default or custom one.